solutions/devsprint-1/DeliveryAppChallenge/Components/Cells/RestaurantCell.swift

```diff
@@ -0,0 +1,86 @@
+//
+//  RestaurantCell.swift
+//  DeliveryAppChallenge
+//
+//  Created by Edson Neto on 04/11/21.
+//
+
+import UIKit
+
+class RestaurantCell: UITableViewCell{
```

Comment:
Tente usar sempre que possível final nas classes garantindo para o compilador que essa classe não prove herança em nenhum caso.
```suggestion
final class RestaurantCell: UITableViewCell{
```

Reply:
Feito